client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -138,9 +150,19 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 		fetchMigrationEnabledStatus();
 	};
 
+	// We want to record the tracks event, so we use the same condition as the one in the render function
+	// This should be better handled by using a state after the refactor
+	useEffect( () => {
+		if ( ! showUpdatePluginInfo && isTargetSitePlanCompatible ) {
+			dispatch(
+				recordTracksEvent( 'calypso_site_migration_ready_screen', migrationTrackingProps )
```

Comment:
This might triggers before the `sourceSiteId` returns from the API and send out an event with `source_site_id` 0. Maybe we want to add the `sourceSiteId` check here?

Reply:
As we agreed during the chat session, I'll remove the `sourceSiteId` from the list of tracking event properties.